notebooks/Binary Classification with the UCI Credit-card Default Dataset.ipynb

```diff
@@ -495,8 +514,8 @@
     "# Train GridSearch\n",
     "sweep = GridSearch(model,\n",
     "                   constraints=EqualizedOdds(),\n",
-    "                   grid_size=41,\n",
-    "                   grid_limit=2)\n",
+    "                   grid_size=50,\n",
+    "                   grid_limit=3)\n",
```

Comment:
Would be nice to elaborate on these choices. 

Also: What prompted the change?

Reply:
To get better coverage of the appropriate Lagrange multiplier values. But it would be better to use exponentiated gradient.